### DISCUSSION TITLE
Windows Cross compiler for oqs-provide

### DISCUSSION THREAD

**Discussion by ayanqnu** [open-quantum-safe/oqs-provider#397] [Q&A]
Is there any Windows cross compiler available for oqs-provide 0.52. Basically I have code base on Ubuntu 22 and looking for i686-w64-mingw32-gcc based cross compiler. If available, from where can I get and how to use.

**Comment by baentsch** [comment]
We presently do not have support for Win-crosscompile but welcome contributions for this, e.g., along the lines shown in [liboqs](https://github.com/open-quantum-safe/liboqs/blob/main/.CMake/toolchain_windows-amd64.cmake).
